Chapters/2-4-Chapter11.md

```diff
@@ -0,0 +1,169 @@
+## Chapter 11
+
+A rustling sound awoke him from his slumber.
+
+"Someone's here," Casra whispered. From underneath their bed where Linyu slept, thi pulled out two steel-tipped wooden poles and handed one to Kipnu on hae way toward the ladder.
+
+"I'll go first," thi said.
+
+Kipnu watched as Casra went up the ladder and cracked the door to peer through. From the corner of his eye, he could see Linyu, awake and cowering in the corner under a blanket.
+
+"Stay here," Casra commanded Kipnu. Thi went up through the door, and Kipnu went up the ladder to watch and listen.
+
+The figure in the dark had an unmistakable spear - Setre was here.
+
+"Casra. You've come back?", she asked in a bewildered tone. "Are you holding Kipnu here? Don't lie to me."
+
+"So what if I am?"
+
+"I captured him. He is property of our tribe. You know this."
+
+"People cannot be owned, Setre. I thought you were better than this."
+
+"I'm not your guardian anymore. You have no idea what happened in the all the time you were gone. Things have changed now."
+
+"When I left, you had such conviction and even more strength than I had. I taught you everything I knew. Why didn't you bring Shinna to justice?"
+
+She rose her voice, as if to say something she'd been waiting to say for a long time.
+
+"I'm not you, Casra. They trusted you with their lives, but me? I'm the one that put you on a boat and sent you away as everything fell apart. I might as well have been Shinna haeself, the way they looked at me after that."
+
+"You sent me away for Linyu's safety. They knew that."
+
+"How could they trust me? I told them you would be back, but you never showed. They had to move on. But I never gave up on you. I've been waiting for you. For so long."
+
+Her voice began to break now that hae presence had sunken in, and she released her spear to the ground, moving close to embrace Casra.
+
+"The things I had to do to survive," she said through tears, "You wouldn't think I was the same person."
+
```

Comment:
I am starting to lose track of who is saying what again, even though they say eachother's name every other sentence.

I think the problem without writing dialog tags is you are giving the reader something extra to do, while they are trying to imagine the conversation. It cost them energy to keep track of who is saying what, and instead of immersing into the story they have to be constantly reminded they are reading because they have to keep track of who said what.

Reply:
It violates the "don't make me think" principle. Art should very much make people think. It should ask philosophical questions, raise awareness, etc. But you are making your reader have to do think about the unimportant stuff such as who said what.